test/px-context-multiselect-tests.js

```diff
@@ -0,0 +1,68 @@
+// This is the wrapper for custom tests, called upon web components ready state
+
+document.addEventListener("WebComponentsReady", function() {
+  runCustomTests();
+});
+function runCustomTests() {
+  var selectContextMS = Polymer.dom(document).querySelector('#px_context_browser_multiselect');
+  var selectContextSA = Polymer.dom(document).querySelector('#px_context_browser_selectall');
+  var selectContextNoSA = Polymer.dom(document).querySelector('#px_context_browser_noselectall');
+  var selectContextLimit = Polymer.dom(document).querySelector('#px_context_browser_limit');
+
+  // This is the placeholder suite to place custom tests in
+  // Use testCase(options) for a more convenient setup of the test cases
+
+  suite('Custom Automation Tests for px-context-browser with multiSelect functionality', function() {
```

Comment:
I would advise adding some additional tests in here to cover the selected/deselection code paths that were introduced: 

* the flow of adding an items to `selectedItems` when its checkbox or open button is clicked, and the flow of removing it when its checkbox is clicked again
* the flow of selecting or deselecting all items in a list when the select all checkbox is tapped